lib/formulator.ex

```diff
@@ -83,16 +93,37 @@ defmodule Formulator do
   defp build_input(form, field, options, error) do
     input_type = options[:as] || :text
     input_class = options[:class] || ""
+
     options =
       options
+      |> add_validation_attributes(form, field)
```

Comment:
What do you think about moving the associated deleting of `:validate` into this function? Seems like this function should handle its own cleanup.